dna_rna_tools/dna_rna_tools.py

```diff
@@ -0,0 +1,60 @@
+def run_dna_rna_tools(*args):
+    *sequence, procedure = args
+
+    if procedure == 'transcribe':
+        return apply_to_sequences(sequence, transcribe)
+    elif procedure == 'reverse':
+        return apply_to_sequences(sequence, reverse)
+    elif procedure == 'complement':
+        return apply_to_sequences(sequence, complement)
+    elif procedure == 'reverse_complement':
+        return apply_to_sequences(sequence, reverse_complement)
+    else:
+        return None
+
+
+def valid_rna(sequence):
+    if set(sequence).issubset({'A', 'U', 'G', 'C', 'a', 'u', 'g', 'c'}):
+        return True
+    else:
+        return False
+
+
+def valid_dna(sequence):
+    if set(sequence).issubset({'A', 'T', 'G', 'C', 'a', 't', 'g', 'c'}):
+        return True
+    else:
+        return False
+
```

Comment:
Здорово что исправила, но тут можно еще вроще. Ровно кстати этот пример мы разбирали на лекции про функции:
```suggestion
def valid_dna(sequence):
    return set(sequence).issubset({'A', 'T', 'G', 'C', 'a', 't', 'g', 'c'}):
```

Не надо проверять логическое значение логического утверждения чтобы вернуть другое логическое значение. Можо сразу возвращать что есть :)